core/templates/dev/head/pages/exploration_editor/simple_editor_tab/simple_editor_body_directive.html

```diff
@@ -131,4 +140,36 @@
     text-align: center;
     text-shadow: -1px 0 white, 0 1px white, 1px 0 white, 0 -1px white;
   }
+
+  simple-editor-body .question-type-wrapper {
+    background-color: #fff;
```

Comment:
Feel free to just use `white` instead of the HTML code, it makes the CSS more readable.